web/docs/command-line.md

```diff
@@ -103,12 +103,12 @@ the following rules:
    include a literal underscore.
 
 From the perspective of the command line, the environment variable key
-`TENZIR_X__Y__Z` maps to `tenzir x y --z`. Here are two examples with identical
-semantics:
+`TENZIR_X__Y__Z` maps to `tenzir-ctl x y --z`. Here are two examples with
+identical semantics:
 
 ```bash
-TENZIR_IMPORT__BATCH_SIZE=42 tenzir import json < data
-tenzir import --batch-size=42 json < data
+TENZIR_IMPORT__BATCH_SIZE=42 tenzir-ctl import json < data
+tenzir-ctl import --batch-size=42 json < data
 ```
```

Comment:
I don't think this is true in general, e.g. `TENZIR_PLUGINS__PLATFORM__API_KEY=xxx` doesn't map to `tenzir-ctl plugins --api-key=xxx`. I'd remove this whole section.

Reply:
Okay, will remove.